src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -10,21 +13,63 @@ namespace gpu {
 
 struct context;
 
-struct miopen_gemm
+template <class Op>
+struct rocblas_gemm
 {
-    op::dot op;
+    Op op;
 
     template <class Self, class F>
     static auto reflect(Self& self, F f)
     {
         return migraphx::reflect(self.op, f);
     }
 
-    std::string name() const { return "gpu::gemm"; }
-    shape compute_shape(const std::vector<shape>& inputs) const;
+    std::string name() const
+    {
+        if(op.name().find("quant_") != std::string::npos)
```

Comment:
you can use contains() for the if condition. Also, you don't need an else statement here and can just return "gpu::gem".

Reply:
fixed